pkg/csi-common/server.go

```diff
@@ -17,21 +17,20 @@ limitations under the License.
 package csicommon
 
 import (
+	"github.com/container-storage-interface/spec/lib/go/csi"
```

Comment:
nit: pls keep sequence of import packages, e.g. first basic package, then github packages.